app/uk/gov/hmrc/customs/declarations/stub/controllers/SecureTwoWayMessagingController.scala

```diff
@@ -370,21 +368,21 @@ object SecureTwoWayMessagingController {
         |   }
         |</style>
         |<div class="govuk-body-l">
-        |   <h1 class="govuk-heading-l margin-top-small margin-bottom-small">about import doc test</h1>
+        |   <h1 class="govuk-heading-l margin-top-small margin-bottom-small">MRN123456789098765 needs some action</h1>
         |   <p>
         |      <span aria-hidden="true" class="custom-caption">
-        |      HMRC Imports(CDS) needs actions sent this on 17 May 2023 at 10:43am
+        |      National Clerence Hub sent this on 3 December 2024 at 3:00pm
```

Comment:
Should it not be 
**National Clearance**

Reply:
Yes. Copied from the DC service in QA. Will update our copy ;)